datamade/openness-project-nmid
# Implement expenditure import

## Overview

This PR extends the contribution import to also import expenditures. It also updates the Makefile recipes to import either type from a local file or S3.

## Testing Instructions

* View the deploy preview and confirm it contains data from before and after 2020.
* Click around and confirm you don't see any Python errors.
* Note in your review any issues you notice. I'll create a bug hunt PR separately.
